File: LeetCode/1470. Shuffle the Array/JavaScript/01. Shuffle the Array.js
var shuffle = function (nums, n) {
    let sortedArray = [];
    let arrayIndex = 0;
    for (let i = 0; i < n; i++) {
        sortedArray[arrayIndex++] = nums[i];
        sortedArray[arrayIndex++] = nums[i + n];
    }
    return sortedArray;
};

File: LeetCode/1470. Shuffle the Array/C#/01. Shuffle the Array.cs
int[] Shuffle(int[] nums, int n)
{
    int[] sortedArray = new int[nums.Length];
    int arrayIndex = 0;
    for (int i = 0; i < n; i++)
    {
        sortedArray[arrayIndex++] = nums[i];
        sortedArray[arrayIndex++] = nums[i + n];
    }
    return sortedArray;
}